Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml

```
@@ -93,3 +93,36 @@
     sprite: DeltaV/Clothing/Uniforms/Jumpskirt/centcom_officer.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpskirt/centcom_officer.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpskirtChiefJustice
+  name: chief justice's jumpskirt
+  description: A fancy black jumpskirt with a lace cravat to make it even more fancy. Proper judicial attire.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/cj.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniform/Jumpskirt/cj.rsi
+    
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpskirtClerk
+  name: clerk's dress skirt
+  description: A modest dress skirt for the person with the power to notarize anything.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/clerk.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/clerk.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpskirtProsecutorRed
+  name: prosecutor's dress skirt
+  description: A red suit and skirt with a fancy cravat. Perfect for a prosecutor.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/prosecutorred.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/prosecutorred.rsi
```

```suggestion
    sprite: DeltaV/Clothing/Uniforms/Jumpskirt/prosecutorred.rsi

```
Some of these .rsi names are making me thing these should start being segmented like prosecutor-red.rsi